src/fklearn/preprocessing/schema.py

```diff
@@ -0,0 +1,134 @@
+import functools
+import inspect
+import pandas as pd
+import toolz
+
+from typing import Any, Callable, Dict, List, Optional, Union
+from fklearn.types import LearnerLogType, LearnerReturnType
+
+
+@toolz.curry
+def feature_duplicator(
+    df: pd.DataFrame,
+    columns_to_duplicate: Optional[List[str]] = None,
+    columns_mapping: Optional[Dict[str, str]] = None,
+    preffix: Optional[str] = None,
+    suffix: Optional[str] = None
+) -> LearnerReturnType:
+    """
+    Duplicates some columns in the dataframe
```

Comment:
Would be cool to provide are clearer docstring, explaining what is happening and giving examples(same for the `column_duplicatable` method)

Reply:
Is there an example on which i can ground?

Reply:
I don't have a decent example, as we don't have a lot of other stuff doing similar things, but it could be more complete like the following:

https://github.com/nubank/fklearn/blob/master/src/fklearn/validation/splitters.py#L264
https://github.com/nubank/fklearn/blob/master/src/fklearn/validation/splitters.py#L389
https://github.com/nubank/fklearn/blob/master/src/fklearn/training/pipeline.py#L13

In your case would be cool to have an example explaining exactly the behaviour, this way people don't need to see this PR to understand what is happening